src/main.ts

```diff
@@ -18,6 +18,8 @@ async function bootstrap() {
     }),
   );
 
+  app.enableCors();
```

Comment:
just a question: why did you need to enable CORS?

Reply:
`Access to fetch at 'http://localhost:3000/auth/reset-password' from origin 'null' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`